src/index.ts

```diff
@@ -1,17 +1,43 @@
-import Clerk from './Clerk';
-import ClerkExpressMiddleware from './middleware/expressjs';
-import {
-  withSession,
-  requireSession,
-  WithSessionProp,
-  RequireSessionProp,
-} from './middleware/nextjs';
+import Clerk from './instance';
+
+// Export as a named export in case the dev wishes to create Clerk instances as well
+export { Clerk };
 
-export default Clerk;
 export {
   ClerkExpressMiddleware,
   withSession,
   requireSession,
   WithSessionProp,
   RequireSessionProp,
-};
+} from './instance';
+
+// Export a default singleton instance that should suffice for most use cases
+const singletonInstance = Clerk.getInstance();
+export default singletonInstance;
```

Comment:
🙃 We could movet his along with the other exports in line 24 to have all the exports in one place.

Reply:
Done